src/sonic-config-engine/sonic-cfggen

```diff
@@ -304,7 +305,12 @@ def main():
             }}}
         # The ID needs to be passed to the SAI to identify the asic.
         if asic_name is not None:
-            hardware_data['DEVICE_METADATA']['localhost'].update(asic_id=asic_id)
+            device_id = get_npu_pci_device_id(asic_id)
+            # if the pci_id obtained is None, exit with error
+            if device_id is None:
+                print('Failed to get pci ID for', asic_name)
```

Comment:
can we add similar to  below and redirect to sys.stderr
print('Failed to get port config', file=sys.stderr)

Reply:
Thanks, Updated to use stderr.